docs/01-Introduction.md

```diff
@@ -53,10 +53,12 @@ study-defined MRI protocol
  - BrainBrowser using 3D or 4D navigation of these MINC files. More details on 
 BrainBrowser's capabilities can be found [here.][5]
 
-`*` Please note that while acquisitions with `localizer` as their series description 
-are included in the DICOM archival step, they are excluded in all the steps that start 
-at, and follow the DICOM to MINC conversion. In a future release, the list of series 
-description to exclude will become customizable by projects.  
+`*` Please note that all acquisitions are included in the DICOM archival
+step (even the ones excluded at a further step of the pipeline). Specific
+acquisitions (such as localizers or scouts) can be excluded from the steps of
```

Comment:
Code format for localizers and scouts

Reply:
1) I would rephrase here to say: "Please note that all acquisitions are included in the DICOM archival step. However, specific acquisitions ...".

2) Is it possible to put the `*` for both items a few lines before (DICOM archive and Imaging Browser)? as this note pertains to both of them.